js/index.js

```diff
@@ -1,7 +1,112 @@
+// let betweenLetters=" ";
+
+// console.log(Hacker1UpperCase.split("").join(betweenLetters));
+//3.2
+//Jamal Nasser
+let reversedName = '';
+for (let i = hacker2.length -1; i >= 0; i--){
+  reversedName += hacker2[i];
+}
+console.log(reversedName);
+
+//Jeroen van Bart
+//let stringReverse = hacker1.split("").reverse().join("");
+//console.log(stringReverse);
+
+//3.3
+//Jamal Nasser
+if (hacker1.localeCompare(hacker2) === 1) {
+  console.log(`The driver's name goes first.`);
+} else if (hacker1.localeCompare(hacker2) === -1) {
+  console.log(`Yo, the navigator goes first definitely.`);
+} else {
+  console.log(`What?! You both have the same name?`);
+}
+
+//jeroen van bart
+// if(hacker1.localeCompare(hacker2) == 1){
+//     console.log("Yo, the navigator goes first definitely.")
+// }else if(hacker1.localeCompare(hacker2) == -1){
+//     console.log("The driver's name goes first.")
+// }else{ console.log("What?! You both have the same name?")};
+
+//Bonus1
+//Jamal Nasser
+let paragraph = `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum. \n It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. Various versions have evolved over the years, sometimes by accident, sometimes on purpose injected humour and the like. \n There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which don't look even slightly believable. If you are going to use a passage of Lorem Ipsum, you need to be sure there isn't anything embarrassing hidden in the middle of text. All the Lorem Ipsum generators on the Internet tend to repeat predefined chunks as necessary, making this the first true generator on the Internet. It uses a dictionary of over 200 Latin words, combined with a handful of model sentence structures, to generate Lorem Ipsum which looks reasonable. The generated Lorem Ipsum is therefore always free from repetition, injected humour, or non-characteristic words etc.`
+
+
```

Comment:
Nice attempt with using a `function`, however, the idea here is to practice with the loops and conditionals you've learned.